polylogue/importers/claude_ai.py

```
@@ -32,6 +34,29 @@ def _load_bundle(path: Path) -> Tuple[Path, Optional[TemporaryDirectory]]:
     return Path(tmp.name), tmp
 
 
+def _iter_conversations(convo_path: Path):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_iter_conversations`

(ANN202)